<!DOCTYPE html>
<html lang="en">
  <head>
    <!-- Google tag (gtag.js) -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-870YFV54KL"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-870YFV54KL');
    </script>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <title>FexSplice about</title>

    <!-- Bootstrap -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

  <body>
  <!-- jQuery (necessary for Bootstrap\'s JavaScript plugins) -->
  <script src="http://code.jquery.com/jquery.js"></script>
  <!-- Include all compiled plugins (below), or include individual files as needed-->
  <script src="js/bootstrap.min.js"></script>

  <!-- header -->
  <div id="header" class="container">
    <nav class="navbar navbar-default">
    <div class="navbar-header">
      <a class="navbar-brand" href=index.html>Top</a>
      <a class="navbar-brand" href=about.html>About</a>
      <a class="navbar-brand" href=download.html>Download</a>
    </div>
    </nav>
  </div>
    
  <div class="container">
    <!-- Main jumbotron for a primary marketing message or call to action -->
    <div class="jumbotron">
      <p>
FexSplice predicts a splicing consequence of a single nucleotide variation (SNV) at the first nucleotide of an exon (Fex-SNV). Fex-SNV affects splicing at AG-dependent 3’ splice sites (ss), in which the polypyrimidine tract (PPT) is short or degenerate, and requires binding of U2AF35 at the intron/exon boundary to reinforced the binding of U2AF65 to PPT. In contrast, Fex-SNV does not affect splicing at AG-independent 3’ ss, in which PPT is long. When the first nucleotide of an exon is not G, binding of U2AF35 to the intron-exon boundary becomes weak, and such 3’ ss’s are mostly AG-independent.<br />
      </p>
      <p>
      FexSplice was generated by LightGBM with 106 splicing-affecting and 106 neutral Fex-SNVs extracted from Human Gene Mutation Database Professional (HGMD Pro) released in April 2020 and ClinVar with CLNSIG = pathogenic released in March 2021, and from dbSNP build 151 on GRCh37/hg19 with minor allelic frequency (MAF) greater than 0.01 and less than 0.5, respectively. A total of 115 features dictating the strength of splicing cis-elements, which included the length and nucleotide composition of the polypyrimidine tract; the composition of the branch point sequence; the nucleotides at intronic positions -6 and -5 and exonic positions +2 and +3 that are critical determinants of splicing in our previous analysis (Shibata A. <i>et al</i>. <i>J Hum Genet</i> 2016, 61, 633-640, doi:10.1038/jhg.2016.23), MaxEntScan scores at the 3' and 5' splice sites (Yeo G. <i>et al</i>. <i>J Comput Biol</i> 2004, 11, 377-394, doi:10.1089/1066527041410418), and the presence of recognition motifs of each RBP according to SpliceAid2 (Piva F. <i>et al</i>. <i>Hum Mutat</i> 2012, 33, 81-85, doi:10.1002/humu.21609). Recursive elimination of features by cross-validation showed that 15 features best predicted the splicing consequences of Fex-SNVs with an area under the curve of the receiver operating characteristics curve of 0.86 ± 0.08 (mean and SD) by 10-fold cross-validation.
      </p>
      <p>
LightGBM automatically generates a probability score for each Fex-SNV with 0.5 being the threshold. The default threshold of 0.5 by LightGBM was used in FexSplice. Fex-SNVs with a probability less than 0.5 are predicted to be splicing-insensitive, while those with a probability of 0.5 or more are predicted to be splicing-affecting.
      </p>
      <p>
Citation:
        <ul>
Joudaki A, Takeda J, Masuda A, Ode R, Fujiwara K, Ohno K. FexSplice: a LightGBM-based model for predicting the splicing effect of a single nucleotide variant affecting the first nucleotide G of an exon. <a href=https://www.mdpi.com/2073-4425/14/9/1765 target=_blank><b><i>Genes.</i></b> 14(9): 1765, 2023.</a> (PMID: <a href=https://pubmed.ncbi.nlm.nih.gov/37761905/ target=_blank>37761905</a>)
        </ul>
      </p>
      <p>
      </p>
    </div>

    <img src="Fig-1.png" title="Figure 1" width="1017" height="474"><br><br>
Figure 1. AG-dependent and AG-independent 3’ splice sites (ss). Introns with a short or degenerate PPT require both U2AF65 and U2AF35 for the recognition of the 3′ ss, which is called the AG‐dependent 3′ ss. Introns with a long stretch of PPT strongly binds to U2AF65 and does not require binding of U2AF35, which is called the AG‐independent 3′ ss. The 3’ ss’s without G at the first nucleotide of an exon are mostly AG‐independent.<br>
    <br><br>

  </div>
    
  <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
  <!-- Include all compiled plugins (below), or include individual files as needed -->
  <script src="js/bootstrap.min.js"></script>
  </body>
</html>

